lms/djangoapps/teams/tests/test_views.py

```diff
@@ -510,14 +518,38 @@ def test_bad_course_data(self, status, data):
         self.post_create_team(status, data)
 
     def test_student_in_team(self):
-        self.post_create_team(
+        response = self.post_create_team(
             400,
-            {
-                'course_id': str(self.test_course_1.id),
-                'description': "You are already on a team in this course."
-            },
+            data=self.build_team_data(
+                name="Doomed team",
+                course=self.test_course_1,
+                description="Overly ambitious student"
+            ),
             user='student_enrolled'
         )
+        self.assertEqual(
+            "You are already in a team in this course.",
+            json.loads(response.content)["user_message"]
+        )
+
+    @ddt.data('staff', 'course_staff', 'community_ta')
+    def test_privileged_create_multiple_teams(self, user):
+        """ Privileged users can create multiple teams, even if they are already in one. """
+        # First add the privileged user to a team.
+        self.post_create_membership(
+            200,
+            self.build_membership_data(user, self.test_team_1),
+            user=user
+        )
+
+        self.post_create_team(
+            data=self.build_team_data(
```

Comment:
Note to reviewers: post_create_team checks the status response of 200 if you don't explicitly pass in an expected response code.